src/main/java/frc/robot/subsystems/swerve/ModuleIOReal.java

```diff
@@ -210,7 +210,7 @@ public void updateInputs(ModuleIOInputs inputs) {
     var samples =
         PhoenixOdometryThread.getInstance()
             .samplesSince(lastUpdate, ImmutableSet.of(drivePosition, turnPosition));
-    if (!samples.isEmpty()) {
+    if (samples.stream().filter(s -> s != null).count() != 0) {
```

Comment:
I don't think this filter should ever apply. The journal never has null values pushed.